UnityProject/Assets/Scripts/Shuttles/ShuttleHeater.cs

```diff
@@ -19,4 +19,32 @@ public override void SetAnchored(bool value)
 	{
 		anchored = value;
 	}
+
+	public override void CalculateDirection()
+	{
+		direction = 0;
+		var rotation = transform.rotation.eulerAngles.z;
+		if (rotation >= 45 && rotation <= 135)
```

Comment:
All of this reminds me of something...
https://github.com/unitystation/unitystation/blob/5aa2bc64d6a191068f618000c86ef218ae11b5c7/UnityProject/Assets/Scripts/Transform/Orientation.cs#L147-L160
Why not use Orientation for this?

Reply:
'cause, you know, you're against code duplication and still used `if (rotation >= 45) {blablabla}` thingy in three places within this PR
I mean I can merge this now, but are you planning to refactor this later?

Reply:
thats the plan